test/data/network/network-1.tsv.data

```diff
@@ -0,0 +1,5 @@
+regulator target attr1 attr2 attr3
```

Comment:
To pass the qunit test.. Because for now the qunit tests don't upload data, only do queries.